scripts/lib/CIME/provenance.py

```diff
@@ -16,21 +16,23 @@ def _get_batch_job_id_for_syslog(case):
     mach_syslog only works on certain machines
     """
     mach = case.get_value("MACH")
-    if mach == 'titan':
-        return os.environ["PBS_JOBID"]
-    elif mach in ['edison', 'cori-haswell', 'cori-knl']:
-        return os.environ["SLURM_JOB_ID"]
-    elif mach == 'mira':
-        return os.environ["COBALT_JOBID"]
-    else:
-        return None
-
-def save_build_provenance_acme(case, lid=None):
+    try:
```

Comment:
It's complicated. I believe it has something to do with only wanting to do operations requiring job-id on certain machines. The ACME performance team is picky on maintaining exact compliance with the pre-CIME5 behavior.